main.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 require_relative 'services/browser'
-require_relative 'services/parser'
+require_relative 'services/parser/parser'
 require_relative 'processors/shop_hodinkee'
```

Comment:
do you really need all these packages here?

Reply:
no, i removed it